ibridgesgui/gui_utils.py

```diff
@@ -56,11 +66,10 @@ def get_downloads_dir() -> pathlib.Path:
         Absolute path to 'Downloads' directory.
 
     """
-    if isinstance(pathlib.Path('~'), pathlib.PosixPath):
-        return pathlib.Path('~', 'Downloads').expanduser()
-    else:
-        import winreg
-        sub_key = r'SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders'
-        downloads_guid = '{374DE290-123F-4565-9164-39C4925E467B}'
-        with winreg.OpenKey(winreg.HKEY_CURRENT_USER, sub_key) as key:
-            return pathlib.Path(winreg.QueryValueEx(key, downloads_guid)[0])
+    # Linux and Mac Download folders
+    if pathlib.Path("~", "Downloads").expanduser().is_dir():
```

Comment:
It is used for the download button in the browser tab of the GUI. When you select an object or collection to download , it will end up in your Downloads folder. The user does not have the choice here to select another destination.
The up and download buttons in the browser were and I think are still not meant for severe data transfers, but only for quick and single file down and uploads. For the severe and lengthy data transfers we will create the synchronisation tab that will also show more details about the transfers.